presto-docs/src/main/sphinx/admin/session-property-managers.rst

```diff
@@ -60,7 +60,7 @@ Consider the following set of requirements:
   limit of 1 hour (tighter than the constraint on ``global``).
 
 * All ETL queries (tagged with 'etl') are routed to subgroups under the ``global.pipeline`` group, and must be
-  configured with certain properties to control writer behavior.
+  configured with certain properties to control writer behavior and a hive catalog property.
```

Comment:
It'll be good to modify the `Limitations` section in this file to be consistent with the new behavior.

Reply:
Done